experiments/AMIP/components/land/bucket_init.jl

```diff
@@ -135,25 +122,41 @@ function surface_air_density(
 end
 
 """
-    ClimaLSM.Bucket.liquid_precipitation(atmos::CoupledAtmosphere, p, t)
+    ClimaLSM.liquid_precipitation(atmos::CoupledAtmosphere, p, t)
 
 an extension of the bucket model method which returns the precipitation
 (m/s) in the case of a coupled simulation.
 """
-function liquid_precipitation(atmos::CoupledAtmosphere, p, t)
+function ClimaLSM.liquid_precipitation(atmos::CoupledAtmosphere, p, t)
     # coupler has filled this in
-    return p.bucket.P_liq
+    return p.drivers.P_liq
 end
 
 """
-    ClimaLSM.Bucket.snow_precipitation(atmos::CoupledAtmosphere, p, t)
+    ClimaLSM.snow_precipitation(atmos::CoupledAtmosphere, p, t)
 
 an extension of the bucket model method which returns the precipitation
 (m/s) in the case of a coupled simulation.
 """
-function snow_precipitation(atmos::CoupledAtmosphere, p, t)
+function ClimaLSM.snow_precipitation(atmos::CoupledAtmosphere, p, t)
     # coupler has filled this in
-    return p.bucket.P_snow
+    return p.drivers.P_snow
+end
+
+function ClimaLSM.initialize_drivers(a::CoupledAtmosphere{FT}, coords) where {FT}
+    keys = (:P_liq, :P_snow)
+    types = ([FT for k in keys]...,)
+    domain_names = ([:surface for k in keys]...,)
+    model_name = :drivers
+    # intialize_vars packages the variables as a named tuple,
+    # as part of a named tuple with `model_name` as the key.
+    # Here we just want the variable named tuple itself
+    vars = ClimaLSM.initialize_vars(keys, types, domain_names, coords, model_name)
+    return vars.drivers
+end
+
+function ClimaLSM.initialize_drivers(a::CoupledRadiativeFluxes{FT}, coords) where {FT}
```

Comment:
todo: remove this function after ClimaLSM v0.6.0 release + update. Maybe we can actually add both these initialize_drivers methods to ClimaLSM? @kmdeck

Reply:
yes! I will start on moving them over this week & ask for review. I am thinking the struct definitions of the `Coupled` types would also move?

Reply:
Yeah, I think they should be moved to src/